src/service/getAllCurrencies.ts

```diff
@@ -0,0 +1,9 @@
+import axios from 'axios'
```

Comment:
Do you want to have one service for one endpoint ?

Reply:
I think I can add folder getAllCurrencies, correct?

Reply:
no